taqueria-vscode-extension/src/lib/helpers.ts

```diff
@@ -398,12 +424,20 @@ export class VsCodeHelper {
 			let pluginName = pluginInfo?.label;
 			if (!pluginName) {
 				pluginName = await this.promptForPluginInstallation(projectDir);
-			}
-			if (!pluginName) {
-				return;
+				if (!pluginName) {
+					return;
+				}
 			}
 			pluginName = await this.getPathForPluginSource(pluginName);
```

Comment:
Would the following be more readable?
```typescript
const pluginName = pluginInfo?.label || await this.promptForPluginInstallation(projectDir);
if (!pluginName) return;
const pathToPlugin = await this.getPathForPluginSource(pluginName);
```
If you don't think it captures the logic better, then we can leave it. I don't have the full context 😅

Reply:
Np! It's just that I know @mweichert prefers `const` over `let` (from functional programming), which I usually resonate with. But I am ok with either. 😃